### ISSUE TITLE
Move BlockChainName to configuration file to support dev on regtest / testnet

### ISSUE THREAD

**Issue by jessopb** [lbryio/chainquery#66]
BlockChainName is currently a const. Can it be in config or --flag?

```
// global/global.go
14 | const BlockChainName = "lbrycrd_main"
```

**Comment by tiger5226** [comment]
Yes, I will add it as a config.

**Comment by tiger5226** [comment]
solved with 84f875b3f68266c7a1d6ad95cdf0d3dfd68b5ce5
